docs/source/releases.rst

```diff
@@ -2,6 +2,36 @@ Release History
 ###############
 
 
+v1.2.0 (2025-05-13)
+===================
+
+API Breaks
+----------
+- Delete the defunct symbolic link at the root of the repo.
+  Old scripts that launch using this are encouraged to either
+  swap to pmpsui/pmps.py or switch to the python -m pmpsui entrypoint.
+
+Features
+--------
+- Allow us to pass --area instead of --macros in pmpsui entrypoint
```

Comment:
My recollection is that this is actually the removal of `--macros` and the addition of `--area`.  If this is right, I think I'll move this to `API Breaks`

Reply:
Oh you're so right

Reply:
To be clear, the only macro worth passing was covered by --area, but yeah it's a clean break
In some ways a clean break was the point, trying to get away from lugging around multiple ways to launch the app (just use the entrypoint)